lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -463,6 +463,15 @@ def add_creditcard(xml, creditcard, currency=nil)
         end
       end
 
+      def add_cdpt_authentication_value(xml, creditcard)
+        xml.tag! :AuthenticationECIInd, nil
```

Comment:
The in the case of Apple Pay, the Payment Token's payment data contains an ECI value that must be passed along for certain network/issuer combinations: https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html

It's not always required, but in some cases it is in order for the transaction to process properly. I think it would be best to pass along our standard `:eci` option from the options hash here.